### PR TITLE
luci-app-adblock: Fix duplicated text

### PR DESCRIPTION
**Pull request by musashino205**
Fixed some issues.

- Fix duplicated text in Lua source.

- Fix syntax error in sv/adblock.po

And synchronized translations.

### CODE REVIEW

File: applications/luci-app-adblock/po/sv/adblock.po [hunk 1]
```diff
@@ -139,7 +139,7 @@ msgid "Restrict interface trigger to certain interface(s)"
 msgstr ""
 
 msgid "Resume adblock"
-msgstr "Återuppta adblock"
+msgstr "ÁEeruppta adblock"
```
Comment (hnyman) [inline applications/luci-app-adblock/po/sv/adblock.po]: This looks wrong.  "Återuppta" is proper Swedish, but "ÁEeruppta" is not.

File: applications/luci-app-adblock/po/sv/adblock.po [hunk 2]
```diff
@@ -156,7 +156,7 @@ msgid "Status"
 msgstr "Status"
 
 msgid "Suspend / Resume adblock"
-msgstr "Upphäv / Återuppta adblock"
+msgstr "Upphäv / ÁEeruppta adblock"
```
Comment (hnyman) [inline applications/luci-app-adblock/po/sv/adblock.po]: Same here. "Återuppta" is proper Swedish, but "ÁEeruppta" is not.

**Comment by musashino205** [comment]
Oops, sorry. I fixed issues you pointed out, thank you!

It seems that there was a problem with character code...

**Comment by hnyman** [comment]
Thanks.

**Pull request merged**